### ISSUE TITLE
Stream stops playing after a few seconds

### ISSUE THREAD

**Issue by aztennenbaum** [iamtalhaasghar/yewtube#80]
## Issue / Suggestion
I am trying to play "lofi hip hop radio - beats to relax/study to", but  it stops playing after a few seconds (haven't tested anything else)

Run `yt` from the commandline and type `/lofi`

when I select one of the videos it works fine, but when I select the stream it stops after a few seconds. This also happens when I manually search for the url with `url 5qap5aO4i9A`

## Your Environment

```
yt --version
yewtube version    : 2.8.4
yt_dlp version     : 2022.05.18
Python version     : 3.6.9 (default, Mar 15 2022, 13:55:28) 
[GCC 8.4.0]
Processor          : x86_64
Machine type       : x86_64
Architecture       : 64bit, ELF
Platform           : Linux-5.4.0-120-generic-x86_64-with-Ubuntu-18.04-bionic
sys.stdout.enc     : UTF-8
default enc        : utf-8
Config dir         : /home/andrew/.config/mps-youtube
dbus               : 1.2.18
glib               : True
env:TERM           : xterm-256color
env:SHELL          : /bin/bash
env:LANG           : en_US.UTF-8
env:LANGUAGE       : en_US

```

Music player:
```

 mpv --version
mpv 0.27.2 (C) 2000-2017 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
ffmpeg library versions:
   libavutil       55.78.100
   libavcodec      57.107.100
   libavformat     57.83.100
   libswscale      4.8.100
   libavfilter     6.107.100
   libswresample   2.9.100

```

**Comment by iamtalhaasghar** [comment]
See #70

**Comment by aztennenbaum** [comment]
It looks like a different issue, since I am already MPV, which is was the fix for #70. Also it didn’t print any error message like what happened in #70, it just stopped playing. 

If you run `url 5qap5aO4i9A` and try to play it, you will see what I mean
